squash/squash/settings/dev.py

```diff
@@ -13,3 +13,5 @@
         'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
     }
 }
+
+BOKEH_URL='http://localhost:5007'
```

Comment:
erg, somehow I missed this comment.  I will combine it with another small patch that is coming.